### DISCUSSION TITLE
Select2 or similar plugin

### DISCUSSION THREAD

**Discussion by vpashovski** [breezefront/community#229] [1. Help] [answered]
Is there any way to use select2 lib with Breeze JS enabled.
Select2 depends on jQuery, this is the issue.
Or is there any other plugin that can be used?
Our current module uses select2 for some select/multiselect fields. Needs to replace that with something. Any suggestion with what?

**Comment by vovayatsyuk** [comment, marked as answer]
We do not use jQuery, but there is a chance that it will work with Cash. If not, consider one of these libs:

 - https://github.com/Choices-js/Choices
 - https://github.com/brianvoe/slim-select

**Reply by vpashovski** [reply]
Thanks,
Choices-js is what I'm looking at now.
